src/shared/04_apim_pagopa_notifications_service.tf

```diff
@@ -0,0 +1,66 @@
+module "apim_pagopa_notifications_service_product" {
+  source = "git::https://github.com/pagopa/azurerm.git//api_management_product?ref=v2.18.3"
+
+  product_id   = "pagoPA"
+  display_name = "pagoPA Notifications Service"
+  description  = "API product for pagoPA Notifications Service"
+
+  api_management_name = local.pagopa_apim_name
+  resource_group_name = local.pagopa_apim_rg
+
+  published             = true
+  subscription_required = true
+  approval_required     = true
+  subscriptions_limit   = 1000
+
+  policy_xml = file("./api_product/_base_policy.xml")
+}
+
+##############################
+## API notifications service ##
+##############################
+locals {
+  apim_pagopa_notifications_service_api = {
+    display_name          = "pagoPA Shared - notifications service API"
+    description           = "API to support notifications service"
+    path                  = "shared/notifications-service"
```

Comment:
For the basepath I would leave only the product name, without the domain reference, so: 

- shared/notifications-service --> notifications-service

what do you think?